yazi-config/src/manager/manager.rs

```diff
@@ -20,6 +20,7 @@ pub struct Manager {
 	pub show_hidden:  bool,
 	pub show_symlink: bool,
 	pub scrolloff:    u8,
+	pub mouse:        bool,
```

Comment:
```suggestion
	pub mouse:        MouseEvents,
```

`MouseEvents` looks like:

```rust
bitflags! {
	#[derive(Clone, Copy, Debug, Default, PartialEq, Eq)]
  #[serde(try_from = "Vec<String>")]
	pub struct MouseEvents: u8 {
		const CLICK  = 0b00001;
		const SCROLL = 0b00010;
		const TOUCH  = 0b00100;
		const MOVE   = 0b01000;
		const DRAG   = 0b10000;
	}
}
```

You can check out 

https://github.com/sxyazi/yazi/blob/28dfe728ab541f71be27f131e985723253bcff6e/yazi-config/src/popup/offset.rs#L13

 for implementing it.